### DISCUSSION TITLE
Rules when adding back an instance that got removed.

### DISCUSSION THREAD

**Discussion by unixfox** [searxng/searx-instances#150] [Ideas]
Currently when the owner of an instance ask to add back an instance that got removed due to a bad uptime, we add back the instance right away and we give him a final second chance.

The issue is that sometimes the instance is added but fail again to keep a good uptime.

In order to avoid this kind of issue I propose to wait 1 week before adding back the instance and during this 1 week the instance should keep an uptime of 95% or 8 hours max of downtime during 1 week.

What do you think? Is this a good idea? Should we instead ask for 90% of uptime during 1 week because 95% is too strict?

Also if this rule is added, I'll have to setup a list of websites that can be manually added on searx-instances-uptime repository.

**Comment by dalf** [comment]
I'm okay with that. 

Also, I can check if we can run check.searx.space more often.

**Comment by unixfox** [comment]
@ononoki1 Related to https://github.com/searxng/searx-instances/issues/165#issuecomment-1214314813, please get familiar with this new rule. Thank you.

**Reply by ononoki1** [reply]
OK.

**Comment by unixfox** [comment]
@dalf @return42 @mrpaulblack @tiekoetter @ononoki1 You can now add website to be tracked for their uptime without adding them to the public list: https://github.com/searxng/searx-instances-uptime/blob/master/.upptimerc-custom.yml

This is useful for enforcing the rule that I wrote in this GitHub discussion.

**Reply by ononoki1** [reply]
I need write access to searxng/searx-instances-uptime repo to add websites to `.upptimerc-custom.yml`.
